index.js

```diff
@@ -465,3 +465,22 @@ function wrapProjectsCards() {
 }
 
 wrapProjectsCards();
+
+function val() {
+  const Inputmail = document.querySelector('#Mail_Input');
+  const messageErr = document.getElementById('error_message');
+
+  if (Inputmail.value !== Inputmail.value.toLowerCase()) {
+    messageErr.textContent = 'Please, use lowercases letters!';
+    messageErr.style.color = 'blue';
+    return false;
+  }
+  messageErr.textContent = 'Well done!';
+  messageErr.style.color = 'green';
+  setTimeout(() => {
+    messageErr.textContent = 'Well done!';
+  }, 3000);
+  return true;
+}
+
+val();
```

Comment:
- [ ] As requested by the previous reviewer, when an empty form is submitted, the message should be in the line of 'you cannot submit an empty form' and not 'well done'. Kindly fix it.